packages/labextension/src/widget.tsx

```diff
@@ -296,7 +296,7 @@ const KernelUsage = (props: {
                 {props.trans.__('Process ID:')} {usage.pid}
               </div>
               <div className="jp-KernelUsage-separator">
-                {props.trans.__('CPU:')} {usage.kernel_cpu}
+                {props.trans.__('HPU:')} {usage.kernel_cpu}
```

Comment:
?

Reply:
Oops....my bad - I spent ages trying to figure out what which bit of code changed which bit of UI...

Once you have a full set of changes, I'll push a complete set

Reply:
Done